crates/ethereum/primitives/Cargo.toml

```diff
@@ -25,6 +25,7 @@ alloy-consensus = { workspace = true, features = ["serde"] }
 alloy-serde = { workspace = true, optional = true }
 alloy-rlp.workspace = true
 alloy-rpc-types = { workspace = true, optional = true }
+alloy-rpc-types-eth.workspace = true
```

Comment:
we can make this optional here and enable with the alloy-compat feature just like alloy-rpc-types